src/styles/components/_detailsBanner.scss

```diff
@@ -15,7 +13,7 @@ $breakpoint-tablet: 820px;
     font-family: Viga, sans-serif;
     font-size: 1.5625rem; // 25px/16
 
-    @media only screen and (min-width: $breakpoint-tablet) {
+    @media (min-width: $breakpoint-increase-sizes) {
```

Comment:
To me, the meaning of $breakpoint-increase-sizes isn't super clear. For this file, could you do something similar to code-block.scss and rename it to `$breakpoint-increase-font-size` or something similar?

Reply:
This actually came out of pairing w/ Zastrow and I had the same feeling--but he pointed out that if you just say "increase sizes" you can address that it increases font size/padding/margin. Some of the media queries do more than one thing. But I did tinker w/ the idea of specifically saying "increase-font-size" if that is all it does. I'm going to go through and see if any are specific to that and rename those more specifically. Thanks Jared! :)

Reply:
Maybe `$bp-scale-up` or something like that? I feel like that gives an idea that everything sort of gets bigger at that size.

Reply:
As is usually the case in development, it depends 😄 
If the breakpoint affects a variety of things, then a blanket name that describes the overall effect makes sense, but if it literally just changes the font size, then a more specific name works too.

Reply:
Thanks, I did this update & pushed the changes. Osmond will review :)